### DISCUSSION TITLE
Requesting a new feature: adding values files in the werf.yaml

### DISCUSSION THREAD

**Discussion by Aleksey-Kalinin** [werf/werf#5853] [General]
### Why
When publishing microservices projects, it is necessary to collect the values of each microservice. 
For example, domain name, contracts, etc.

### For example
werf.yaml
```
deploy:
  values:
    - microservice/auth/auth.yaml
    - microservice/shell/shell.yaml
```

**Comment by alexey-igrychev** [comment]
Thank you for your request. We're planning to implement dotenv file support soon to meet these needs, allowing you to define values for werf command options for all or/and specific environments.

**Comment by Aleksey-Kalinin** [comment]
This is very good news. Unfortunately, this will not solve my problem, because I would like to be able to download the YAML files of all microservices, which are listed in the initial YAML file, which is now transmitted via env.
